addons/account/models/account_move.py

```diff
@@ -1104,6 +1104,8 @@ def _query_get(self, domain=None):
         if context.get('date_from'):
             if not context.get('strict_range'):
                 domain += ['|', (date_field, '>=', context['date_from']), ('account_id.user_type_id.include_initial_balance', '=', True)]
+            elif context.get('initial_bal'):
+                domain += [(date_field, '<', context['date_from'])]
             else:
```

Comment:
In general ledger report and partner ledger report, there is one field 'initial_balance' which allow you to add a row to display amount of debit/credit/balance that precedes the date_from you have set so that need to add initial_bal condition.